Bottom: 512c2ca031d234e10235909a58a12bb9ca94a18d
Top:    9cb18b0152e07aa6e793551ca74d71f4c2cf6fd3
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2009-10-20 20:09:17 +0100

Refresh of 0001-Additional-defines-for-a-3D-joystick-for-older-lin.patch

---

diff --git a/src/hid/gtk/snavi.c b/src/hid/gtk/snavi.c
index f0494bf..618be32 100644
--- a/src/hid/gtk/snavi.c
+++ b/src/hid/gtk/snavi.c
@@ -21,8 +21,8 @@
 
 #include <linux/types.h>
 #include <linux/input.h>
-/* 
-Additional defines for a 3D joystick for (older) linux versions which do not 
+/*
+Additional defines for a 3D joystick for (older) linux versions which do not
 have these defines in <linux/input.h>
 */
 #ifndef REL_RX
